R/dettl.R

```diff
@@ -1,3 +1,21 @@
+##' Manage data import.
```

Comment:
I think you could add pointers here to the top-level functions that people might want to use instead, or give some guidance about when they should look here vs at those functions

Reply:
I've added some notes about when you'd want to run using the object itself. When the PR with top level functions is merged I will add links to this documentation to those functions and expand the note.